Cork/Models/Packages/Brew Package.swift

```diff
@@ -18,16 +18,11 @@ struct BrewPackage: Identifiable, Equatable
     let versions: [String]
     
     let sizeInBytes: Int64?
-    
-    func convertDateToPresentableFormat(date: Date) -> String
-    {
-        let dateFormatter = DateFormatter()
-        dateFormatter.dateFormat = "d MMM y (E), HH:mm"
+}
 
-        return dateFormatter.string(from: date)
-    }
-    func convertSizeToPresentableFormat(size: Int64) -> String
-    {
-        return convertDirectorySizeToPresentableFormat(size: size)
+extension FormatStyle where Self == Date.FormatStyle {
+    
+    static var packageInstallationStyle: Self {
+        Self.dateTime.day().month(.wide).year().weekday(.wide).hour().minute()
```

Comment:
This is slightly different to the old format, but I thought that there was the space to have a more full date. 

Note that order doesn't matter here – the system localises the components you request into the right format, with the right punctuation

Reply:
That's fair, you're right that there's plenty of space.